### DISCUSSION TITLE
how can I remove single layer with removal mode

### DISCUSSION THREAD

**Discussion by yangbinlovegis** [geoman-io/leaflet-geoman#842] [Questions & Help]
how can I remove single layer or specified layerGroup with removal mode

**Comment by Falke-Design** [comment]
Today there is no easy way to do this, there is already a feature request for this but I don't know when we will implement this. #583

You can use this workaround: https://github.com/geoman-io/leaflet-geoman/issues/583#issuecomment-724133871

Remove the `pm` property for all layers they should not be removable and add it again if you want to make it editable
